CAIPs/caip-27.md

```diff
@@ -57,15 +59,23 @@ The application would interface with a provider to make request as follows:
 }
 ```
 
-The JSON-RPC method is labelled as `caip_request` and expects three parameters:
+The JSON-RPC method is labelled as `caip_request` and expects 
+three **required parameters** and 
+one *optional parameter*:
 
-- chainId - [CAIP-2][]-defined `chainId` to identify both a namespace and a
-  specific chain or network within it
-- session - [CAIP-171][] `SessionToken` to identify the session opened or
+- **session** - [CAIP-171][] `SessionToken` to identify the session opened or
   updated by a [CAIP-25][] interaction.
-- request - an object containing the fields:
-  - method - JSON-RPC method to request
-  - params - JSON-RPC parameters to request
+- **scope** - a `scopeObject` authorized by a [CAIP-25][] response and persisted in
+  the session by both caller and respondent
+- *chainId* - [CAIP-2][]-defined `chainId` including both namespace and a
```

Comment:
I think we should remove this and update the scope definition to be either a non-blockchain scope or a CAIP-2 `chainId`, because the request will have to either map to a specific chain or not, never just a namespace.

Reply:
Not exactly never-- the `wallet_` namespace within eip155 and many methods in polkadot and cosmos ARE namespace-wide!

Reply:
ok fair, but you still wouldn't need the `chainId`, the `scope` would then just be the namespace.